app/models/planet.py

```diff
@@ -0,0 +1,30 @@
+from app import db
+
+class Planet(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    description = db.Column(db.String)
+    color = db.Column(db.String)
+    
+    def to_json(self):
+        return {"id": self.id,
+                "name": self.name,
+                "description": self.description,
+                "color": self.color
+                }
+
+    def update(self, req_body):
+        
+        self.name = req_body["name"]
+        self.description = req_body["description"]
+        self.color = req_body["color"]
+
+    @classmethod
+    def create(cls, req_body):
+        new_planet = cls(
+            name=req_body['name'],
+            description=req_body['description'],
+            color=req_body['color']
+        )
+
+        return new_planet
```

Comment:
This comment applies to your create() and update() methods:

How would you add in validation to make sure that all the required fields are sent in the request to your server?

For example, if someone sent a POST request but left off color, then line 27 would throw KeyError that it couldn't find color.

it would be nice to handle the error and return a message so that the client knows their request was invalid and they need to include color. Something to think about for Task List.